content/webapp/views/components/HTMLSerializers/index.tsx

```diff
@@ -107,12 +111,33 @@ export const defaultSerializer: JSXFunctionSerializer = (
 
       const isInPage = linkUrl.match(/^https:\/\/(#.*)/i);
       const hashLink = isInPage && isInPage[1];
+      const isWorkLink = linkUrl.match(
+        /^https:\/\/wellcomecollection.org\/works/i
+      );
 
       const fileExtension = linkUrl.match(/\.[0-9a-z]+$/i);
 
       const documentType =
         fileExtension && fileExtension[0].substring(1).toUpperCase();
 
+      if (isWorkLink) {
+        return (
+          <div
+            data-id="work-link-component"
+            className="spaced-text-reset"
+            style={{ display: 'inline-flex', alignItems: 'center' }}
+          >
+            <WorkIcon
+              src="https://i.wellcomecollection.org/assets/icons/favicon-32x32.png"
+              alt=""
+            />
+            <WorkLink className="link-reset spaced-text-reset" href={linkUrl}>
```

Comment:
The `spaced-text-reset` class is there twice (on the parent div as well), is that needed?